src/app/pages/home/activity/transaction-details/transaction-details.page.html

```diff
@@ -0,0 +1,31 @@
+<mat-toolbar color="primary" *transloco="let t">
+  <button routerLink=".." routerDirection="back" mat-icon-button>
+    <mat-icon>arrow_back</mat-icon>
+  </button>
+  <span>{{ t('transactionDetails') }}</span>
+</mat-toolbar>
+<div *transloco="let t" class="page-content">
+  <h4 class="top-time">{{ (details$ | async)?.created_at | date: 'short' }}</h4>
+  <mat-card class="transaction-card">
+    <mat-card-content class="row">
+      <mat-label> {{ (details$ | async)?.asset.id }}</mat-label>
+      <button
+        [class]="(details$ | async)?.status.__zone_symbol__value"
+        mat-stroked-button
+        disableRipple
+      >
+        {{ (details$ | async)?.status.__zone_symbol__value }}
```

Comment:
Do not use private properties from Angular, such as `__zone_symbol__value`. You should get the data asynchronously or resolve it first as `status` is a `Promise`.

Also, translate it.